include/event2/dns.h

```diff
@@ -202,6 +202,8 @@ extern "C" {
  * - attempts:
  * - randomize-case:
  * - initial-probe-timeout:
+ * - max-probe-timeout:
+ * - probe-backoff-factor:
```

Comment:
Could you also please describe them?

Reply:
I added more comments in `evdns.c`, should I also add them here

Reply:
This file is used for documentation generation, so let's describe them here too.